openpype/hosts/nuke/api/actions.py

```diff
@@ -20,33 +20,31 @@ class SelectInvalidAction(pyblish.api.Action):
 
     def process(self, context, plugin):
 
-        try:
-            import nuke
-        except ImportError:
-            raise ImportError("Current host is not Nuke")
-
-        errored_instances = get_errored_instances_from_context(context,
-                                                               plugin=plugin)
+        # Get the errored instances for the plug-in
+        errored_instances = get_errored_instances_from_context(
+            context, plugin)
 
         # Get the invalid nodes for the plug-ins
         self.log.info("Finding invalid nodes..")
-        invalid = list()
+        invalid_nodes = set()
```

Comment:
Are we entirely sure that Nuke nodes are hashable? I'm not familiar with their API.